terraform/modules/happy-tfe-okta-service-account/ssm_writer.tf

```diff
@@ -4,7 +4,7 @@ locals {
 }
 
 module "params" {
-  source  = "github.com/chanzuckerberg/cztack//aws-ssm-params-writer?ref=v0.43.1"
+  source  = "git@github.com:chanzuckerberg/cztack//aws-ssm-params-writer?ref=tsmith_aws_provider"
```

Comment:
👍🏻 make sure to update this after the other one merges through. I'll approve for now

Reply:
```suggestion
  source  = "git@github.com:chanzuckerberg/cztack//aws-ssm-params-writer?ref=v0.53.2"
```